animation_nodes/nodes/gpencil/gp_stroke_from_spline.py

```diff
@@ -0,0 +1,34 @@
+import bpy
+from ... base_types import AnimationNode, VectorizedSocket
+from ... data_structures import GPStroke, VirtualFloatList
+
+class GPStrokeFromSplineNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GPStrokeFromSplineNode"
+    bl_label = "GP Stroke From Spline"
+    codeEffects = [VectorizedSocket.CodeEffect]
+
+    useSplineList: VectorizedSocket.newProperty()
+
+    def create(self):
+        socket = self.newInput(VectorizedSocket("Spline", "useSplineList",
+            ("Spline", "spline"), ("Splines", "splines")))
+        socket.defaultDrawType = "PROPERTY_ONLY"
+
+        self.newOutput(VectorizedSocket("GPStroke", "useSplineList",
+            ("Stroke", "stroke"),
+            ("Strokes", "strokes")))
+
+    def getExecutionCode(self, required):
+        return "stroke = self.strokesFromSplines(spline)"
+
+    def strokesFromSplines(self, spline):
+        if spline is None: return GPStroke()
+
+        vertices = spline.points
+        amount = len(vertices)
+        pressures = spline.radii
```

Comment:
Those need to be copied.

Reply:
Omar, is there any reason we need to copy the list?

Reply:
Because if we don't copy, the list will be shared between the spline and the stroke, and edits to one of them will affect the other.